pkg/cmd/roachtest/roachtestutil/mixedversion/mixedversion.go

```diff
@@ -839,6 +864,7 @@ func (t *Test) runCommandFunc(nodes option.NodeListOption, cmd string) stepFunc
 // only available on v22.2.0+.
 func (t *Test) choosePreviousReleases() ([]*clusterupgrade.Version, error) {
 	skipVersions := t.prng.Float64() < t.options.skipVersionProbability
+
```

Comment:
Nit: unintended or you really prefer a line break here? :)

Reply:
Fixed :)